Backend/DataAccessLayer/Migrations/201903090835106_create-table-passwordreset.cs

```diff
@@ -3,32 +3,33 @@ namespace DataAccessLayer.Migrations
     using System;
     using System.Data.Entity.Migrations;
     
-    public partial class PasswordReset : DbMigration
+    public partial class createtablepasswordreset : DbMigration
     {
         public override void Up()
         {
             CreateTable(
-                "dbo.ResetIDs",
+                "dbo.PasswordResets",
                 c => new
                 {
-                    PasswordResetID = c.Guid(nullable: false),
+                    Id = c.Guid(nullable: false),
                     ResetToken = c.String(nullable: false),
                     UserID = c.Guid(nullable: false),
                     ExpirationTime = c.DateTime(nullable: false, precision: 7, storeType: "datetime2"),
                     ResetCount = c.Int(nullable: false),
                     Disabled = c.Boolean(nullable: false),
                     AllowPasswordReset = c.Boolean(nullable: false),
                 })
-                .PrimaryKey(t => t.PasswordResetID)
+                .PrimaryKey(t => t.Id)
                 .ForeignKey("dbo.Users", t => t.UserID, cascadeDelete: true)
                 .Index(t => t.UserID);
+
         }
-        
+
         public override void Down()
         {
-            DropIndex("dbo.ResetIDs", new[] { "UserId" });
-            DropForeignKey("dbo.ResetIDs", "UserId", "dbo.Users");
-            DropTable("dbo.ResetIDs");
+            DropIndex("dbo.PasswordResets", new[] { "UserId" });
+            DropForeignKey("dbo.PasswordResets", "UserId", "dbo.Users");
+            DropTable("dbo.PasswordResets");
         }
     }
 }
```

Comment:
This migration really shouldn't be modified. A new migration should be created that changes the name of that column and updates the primary key.

Reply:
Remember, migrations are like git history. We try not to modify them since it'll break everyone elses databases.